package/rmt-server.spec

```diff
@@ -181,8 +181,8 @@ rm -f %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/extensions/*/*/*/mkmf.log
 find %{buildroot}%{lib_dir}/vendor/bundle/ruby/*/gems/yard*/ -type f -exec chmod 644 -- {} +
 
 # lockfile location
-mkdir -p /run/lock/rmt/
-chown -R _rmt /run/lock/rmt/
+mkdir -p /run/lock/rmt.lock
```

Comment:
you don;t need this anymore, since it's not a `dir` :)

Reply:
oh.. exactly. will have to recheck it while installing package

Reply:
fixed